test/functorch/test_aotdispatch.py

```diff
@@ -2463,6 +2466,7 @@ def fn(x: torch.Tensor, x1: torch.Tensor) -> torch.Tensor:
         # Not checking equality of ref and x as Exception is expected
 
     # Partially addresses https://github.com/pytorch/pytorch/issues/106457
+    @skipIfTorchDynamo()
```

Comment:
it sounds like prior to this PR, this test would work properly under dynamo, but now it does not. Why?

Reply:
Hmm if there answer is because dynamo blows up when trying to run directly on the new custom schema objects that we branch on at runtime, then I agree a skip here seems fine (it is unnecessary to get dynamo working on that). But I'd like a comment next to this @skip explaining exactly what we are not supporting in dynamo

Reply:
There is an error on symbolic shapes guard verbose printing, that appeared after tangents processing change:

https://gist.github.com/IvanKobzarev/339f6b0b1465de56731cb6d6d14f2a9f